megamek/src/megamek/common/options/StaticGameOptions.java

```diff
@@ -0,0 +1,88 @@
+package megamek.common.options;
+
+import megamek.server.victory.BVDestroyedVictoryCondition;
+import megamek.server.victory.BVRatioVictoryCondition;
+import megamek.server.victory.EnemyCmdrDestroyedVictory;
+import megamek.server.victory.KillCountVictory;
+
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class StaticGameOptions implements IGameOptions {
```

Comment:
Improve performance for auto-resolve scenario simulations by not having to load the same set of options hundreds of times.